Snowdog_Components/components/Organisms/order-summary/_order-summary.scss

```diff
@@ -158,5 +158,8 @@
     &__product-price {
         font-size: $order-summary__product-price-font-size;
     }
-}
 
+    .zoid-outlet {
+        z-index: 0;
+    }
```

Comment:
please move it to theme styles and create variable based on global z-index variable